### ISSUE TITLE
Update to Angular 15

### ISSUE THREAD

**Issue by Jakob-em** [Jakob-em/storybook-addon-angular-router#15]

**Comment by Ulse81** [comment]
Hey @Jakob-em,

will there be a version with angular 15-support soon? Are you already working on it or anyone else?

**Comment by Jakob-em** [comment]
Thanks for the reminde @Ulse81, I totally forgot about the update 😅.
I just released version  1.7.0 with support for angular 15 ✅

**Comment by freddycoder** [comment]
Big thank you for the update! I think there is a little typo in the new version. I got this error:
```
npm ERR! Found: @angular/router@14.2.12
npm ERR! node_modules/@angular/router
npm ERR!   @angular/router@"^14.2.12" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/router@"^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 | ^15.0.0" from storybook-addon-angular-router@1.7.0
npm ERR! node_modules/storybook-addon-angular-router
npm ERR!   dev storybook-addon-angular-router@"^1.7.0" from the root project
```

note the single pipe between 14.0.0 and 15.0.0

**Comment by Jakob-em** [comment]
Fixed with 1.7.1 🙂

**Comment by Ulse81** [comment]
Thank you for the extremely prompt response!!
